### PR TITLE
add norm function to jax frontend

### PR DESCRIPTION
**Pull request by mohame54**
#5758

### CODE REVIEW

File: ivy_tests/test_ivy/test_functional/test_extensions.py [hunk 1]
```diff
@@ -165,8 +165,7 @@ def test_sinc(
 @given(
     dtype_and_x=helpers.dtype_and_values(
         available_dtypes=helpers.get_dtypes("float", full=False),
-        min_num_dims=1,
-        max_num_dims=1,
+        min_num_dims=1, max_num_dims=1
```
Comment (aryans1204) [inline ivy_tests/test_ivy/test_functional/test_extensions.py]: Please remove this line and stick to the previous standard of representing function arguments, with one argument on one line, ended by a comma.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py [hunk 2]
```diff
@@ -402,3 +402,58 @@ def test_jax_numpy_solve(
...
+        num_positional_args,
+        native_array,
+):
+    dtype, x, ord_param, axis, keepdims = params
+
+    helpers.test_frontend_function(input_dtypes=dtype,
+                                   as_variable_flags=as_variable,
+                                   with_out=False,
+                                   num_positional_args=num_positional_args,
+                                   native_array_flags=native_array, frontend='jax',
```
Comment (aryans1204) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py:454]: Again, for lint formatting, please only have on functional argument on one line, followed by a comma to adhere to the standard

**Review by aryans1204 — CHANGES_REQUESTED** [review]

**Comment by mohame54** [comment]
OH i see, my baaaad

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py (hunk 2 above)
Comment (mohame54) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py:454]: Thanks for saving me, I didn't what was going wrong

**Review by aryans1204 — APPROVED** [review]
LGTM!

**Comment by mohame54** [comment]
What do you mean ?

**Comment by aryans1204** [comment]
Looks Good To Me(LGTM), it's a common abbreviation to approve merge. I've merged the branch, thanks for contributing!

**Comment by mohame54** [comment]
Oh thank you

**Comment by mohame54** [comment]
So i have a question, this was  a part of my coding challenge, what is the next step ?

**Comment by aryans1204** [comment]
Since this pull request was merged, you would need to paste the link of this PR into the form that would've been sent earlier, and from there the Ivy team will contact you. However, I would recommend you to check out Ivy's discord channel here https://discord.com/channels/799879767196958751/939838320547090482 for more information on the hiring process. Good luck!

**Comment by mohame54** [comment]
thanks for the Help!

**Pull request merged**